### ISSUE TITLE
Show Us Your Now Docks !!!!

### ISSUE THREAD

**Issue by psifidotos** [psifidotos/nowdock-plasmoid#29]
Give the users  a way to show their use case for community inspiration !!!!

**Comment by psifidotos** [comment]
![spectacle j11686](https://cloud.githubusercontent.com/assets/2216474/18294573/f2d95390-74a2-11e6-963a-54c2fd5d4695.png)

my current configuration... activity pager, my photos, now dock plasmoid, notes, clock and weather ! :)

**Comment by SteveKej** [comment]
![image](https://cloud.githubusercontent.com/assets/1503819/18295132/c4fa8982-7496-11e6-9101-f1b3f4b13ff4.png)
Activity Pager, Desktop Pager, Application Menu, Folder View (to some custom scripts), Now Dock, System Tray (modified to use small icons), Weather Widget, Digital Clock, Key State.

**Comment by alexjp** [comment]
![screenshot_20160907_084855](https://cloud.githubusercontent.com/assets/85806/18304065/25bc3130-74d8-11e6-8474-32120998f6c1.png)

Kickoff launcher, nowdock, spacer, systray. Dolphin and Konsole are all activity launchers, others are activity based.

**Comment by psifidotos** [comment]
Beautiful !!!!

**Comment by alexjp** [comment]
I am not going to use a bottom middle panel while this problem exists: 

![screenshot_20160907_142000](https://cloud.githubusercontent.com/assets/85806/18313421/a3d859b2-7506-11e6-97d5-fdd06133df26.png)

That activity bar allignment/cutoff just erks me to no end !

**Comment by psifidotos** [comment]
alex, please lets keep the conversations clean, based on the topic... 

Why not open a "Regression" in VDG Forum in order for the users to show things that the users do not like and the VDG to have a point??? 

Can you please remove this post ?

thanks, in advance!!!

**Comment by alexjp** [comment]
Yes, of course, removed it. Sorry, i thought this was the issue to talk about ideas and opinions on stuff about nowdock.

**Comment by psifidotos** [comment]
No worries...

The issue you reported annoys me also a lot!!! :)

It would be better if you were opening it in VDG forum,
it isnt only Now Dock related, it is also Plasma related and important in my opinion!

**Comment by psifidotos** [comment]
I prefer to keep this as more positive as possible!!! :)

Now Dock isnt there yet, but we will try... :)

**Comment by psifidotos** [comment]
Folder plasmoid in Now Dock Panel showing the current activity's pinned files!!! :)
I love it !!! At last!! Activities showing their love !!! :)

![spectacle ns6828](https://cloud.githubusercontent.com/assets/2216474/18617582/661876f6-7ddb-11e6-9872-9157cd530281.png)

**Comment by High-Flyer** [comment]
![psfi](https://cloud.githubusercontent.com/assets/12442704/18889532/01ff14d0-851b-11e6-812c-b9f7b80e58e7.png)
![psfi2](https://cloud.githubusercontent.com/assets/12442704/18889531/01fd736e-851b-11e6-9e3d-b57c889e7c10.png)
My setup as any plasma widget can be used have infinite possibilities..
Again thank you for great widget and panel.

**Comment by AndydeCleyre** [comment]
This is my work machine right now. Wallpaper is a slideshow. No zoom effects. I don't know why the dbeaver icon is broken. The `.desktop` launcher specifies `Icon=dbeaver` and I've got `/usr/share/icons/papirus/{48x48,32x32,22x22,16x16}/apps/dbeaver.svg`.

EDIT: Oh, it's not just the icon, the dbeaver launcher doesn't work at all. I can add a good looking, and working, launcher by dragging it in, but then the window gets that white icon anyway, in addition to the launcher.

![screenshot_20160929_161623](https://cloud.githubusercontent.com/assets/1787385/18970724/6a1135b2-8660-11e6-895d-e1b2f97713cb.png)

**Comment by psifidotos** [comment]
@AndydeCleyre if you disable shadows does the white circle remains? which version of the plasmoid are u using?

**Comment by AndydeCleyre** [comment]
@psifidotos I'm using latest master panel and plasmoid -- but I found that the icon is corrected if I copy `dbeaver.desktop` to `~/.local/share/applications/` and add the line `StartupWMClass=SWT`.

**Comment by psifidotos** [comment]
@AndydeCleyre nice to hear!!!

**Comment by Steinhagen** [comment]
@psifidotos Congrats for making it into the Plasma 5.8 release video!
https://youtu.be/LgH1Clgr-uE?t=52s

**Comment by psifidotos** [comment]
@Steinhagen hehe.. :) nice to see !!!
